03-visualization.Rmd

```diff
@@ -673,9 +719,16 @@ ggplot(data = weather, mapping = aes(x = factor(month), y = temp)) +
   geom_boxplot()
 ```
 
-We have introduced a new function called `factor()` here.  One of the things this function does is to convert a discrete value like `month` (1, 2, ..., 12) into a categorical variable.  The "box" part of this plot represents the 25^th^ percentile, the median (50^th^ percentile), and the 75^th^ percentile.  The dots correspond to *outliers*.  (The specific formulation for these outliers is discussed in Appendix \@ref(appendixA).)  The lines show how the data varies that is not in the center 50% defined by the first and third quantiles.  Longer lines correspond to more variability and shorter lines correspond to less variability.
+We have introduced a new function called `factor()` here which converts a discrete numerical values, like `month` whose values are the numbers 1 through 12, into a categorical variable variable with "labels" 1 through 12. So for example, after converting `month` to a categorical variable, we couldn't perform any mathematical operations on its values. Note also that the labels have an ordering on the x-axis in Figure \@ref(fig:monthtempbox); this ordering is based on the alphanumeric ordering of the numbers 1 through 12. 
```

Comment:
Not sure this wording makes more sense. It seems a little too wordy and run-on? Let's revisit cleaning it up together.

Reply:
Feel free to break up my run-ons. I do tend to favor run-ons the first time I write something, but then break them up in the polishing phase